ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/activity/TutorialActivity.kt

```diff
@@ -47,13 +47,13 @@ class TutorialActivity : AppCompatActivity(), TutorialCallbacks {
     private var tutorialMenu: Menu? = null
 
     override fun onCreateOptionsMenu(menu: Menu?): Boolean {
-        if (pageSet == PageSet.ONBOARDING) {
-            menuInflater.inflate(R.menu.tutorial_menu, menu)
+        pageSet.menu?.let {
+            menuInflater.inflate(it, menu)
             tutorialMenu = menu
             setMenuVisibility(false)
-        } else {
-            setHomeLinkVisibility(true)
+            return super.onCreateOptionsMenu(menu)
```

Comment:
@wrandall22 `setHomeLinkVisibility(true)` is only being called if there is no menu in the enum.  The `return` will end the call.

Reply:
Yeah, sorry about that, didn't see the return statement at first.